workloads/kube-burner-ocp-wrapper/metrics-profiles/mc-metrics.yml

```diff
@@ -16,3 +16,84 @@
 
 - query: (node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes) and on (instance) label_replace(kube_node_role{role="worker"}, "instance", "$1", "node", "(.+)")
   metricName: mgmt-nodeMemoryUtilization-Workers
+
+- query: (max((sum(irate(node_cpu_seconds_total{}[2m])) by (mode,instance) and on (instance) label_replace(bottomk(1, min_over_time(sum(irate(node_cpu_seconds_total{mode=~"idle",instance!~"{{.MGMT_NON_WORKER_NODES}}"}[2m])) by (mode,instance)[5m:2m] @ {{.Q_TIME}})), "instance", "$1", "instance", "(.+)"))) by (mode, instance)) > 0
+  metricName: mgmtNodeCPU-AggregatedWorkers
+
+- query: bottomk(1,min_over_time(node_memory_MemAvailable_bytes{instance!~"{{.MGMT_NON_WORKER_NODES}}"}[5m:2m] @ {{.Q_TIME}}))
+  metricName: mgmtNodeMemoryAvailable-AggregatedWorkers
+
+- query: (avg(node_memory_MemTotal_bytes{instance!~"{{.MGMT_NON_WORKER_NODES}}"}) by (instance))
+  metricName: mgmtNodeMemoryTotal-AggregatedWorkers
+
+- query: (avg((sum(irate(node_cpu_seconds_total{}[2m])) by (mode,instance) and on (instance) label_replace(cluster:nodes_roles{label_node_role_kubernetes_io_master!=""}, "instance", "$1", "node", "(.+)"))) by (mode, instance)) > 0
+  metricName: mgmtMasterCPU-AggregatedWorkers
```

Comment:
Isn't this metric already covered by `mgmt-nodeCPU-Workers` ?

Reply:
yes, we cannot reuse the other metric as that returns all worker node information and makes it harder to ready from the chart.
I will rename it to be relevant.